libs/portal-layout-styles/src/styles/primeng/sass/theme/extensions/_multiselect.scss

```diff
@@ -89,6 +89,26 @@
   }
 }
 
+.p-multiselect-panel .p-multiselect-items .p-multiselect-item {
+  &:not(.p-disabled).p-focus {
+    background: var(--input-list-item-hover-bg);
+    border-color: inherit;
+  }
+}
+
+.p-multiselect-panel .p-multiselect-header .p-multiselect-close {
+  &:not(.p-disabled).p-focus {
+    background: var(--input-list-item-hover-bg);
+    border-color: inherit;
+  }
+}
+
+.p-multiselect-panel .p-multiselect-header .p-multiselect-close:focus {
+  outline: 1px solid var(--primary-color);
+  outline-offset: 0;
+  box-shadow: none;  
+}
```

Comment:
same thing to consider as above comments

Reply:
changed it to $multiSelectCloseIconFocusOutline